mautrix/bridge/user.py

```diff
@@ -137,3 +145,28 @@ async def push_bridge_state(self, state_event: BridgeStateEvent, error: Optional
         self._prev_bridge_status = state
         await state.send(self.bridge.config["homeserver.status_endpoint"],
                          self.az.as_token, self.log)
+
+    async def send_remote_checkpoint(
+        self,
+        status: MessageSendCheckpointStatus,
+        event_id: EventID,
+        room_id: RoomID,
+        event_type: EventType,
+        message_type: MessageType,
+        error: Optional[Exception] = None,
+    ):
+        await MessageSendCheckpoint(
```

Comment:
Should this have a check for `message_send_checkpoint_endpoint` being set?

```suggestion
        if not self.bridge.config["homeserver.message_send_checkpoint_endpoint"]:
            return
        await MessageSendCheckpoint(
```